assets/js/script.js

```diff
@@ -58,10 +62,9 @@ const brewList = {
     "ratio": 17.42
   }
 }
-
 [espressoEl, coldBrewEl, dripEl, aeroEl, frenchEl, pourEl].forEach((element) => {
   element.click((e) => {
-    grindEl.text(brewList[element.attr('id')].grind-size)
+    grindEl.text(brewList[element.attr('id')].grind - size)
```

Comment:
This syntax is no good, but it looks broken anyway. We will revist in the next PR.